sequencer/src/context.rs

```diff
@@ -241,5 +278,28 @@ async fn handle_events<Ver: StaticVersionType>(
 
         // Generate state signature.
         state_signer.handle_event(&event).await;
+
+        // Send the event via the event streaming service
+        if let Some(events_streamer) = events_streamer.as_ref() {
+            events_streamer.write().await.handle_event(event).await;
+        }
     }
 }
+
+// run an api service to serve internal hotsthot events to external clients (e.g builders)
+async fn run_hotshot_event_streaming_api(url: Url, source: Arc<RwLock<EventsStreamer<SeqTypes>>>) {
+    // Start the web server.
+    let hotshot_events_api = hotshot_events_service::events::define_api::<
+        Arc<RwLock<EventsStreamer<SeqTypes>>>,
+        SeqTypes,
+        Version01,
+    >(&EventStreamingApiOptioins::default())
+    .expect("Failed to define hotshot eventsAPI");
+
+    let mut app = App::<_, EventStreamApiError, Version01>::with_state(source);
+
+    app.register_module("hotshot_events", hotshot_events_api)
+        .expect("Failed to register hotshot events API");
+
+    spawn(app.serve(url, STATIC_VER_0_1));
```

Comment:
We should be using `ctx.spawn` here so we don't leak the task

Reply:
d1ba612c74d0131e4b06907bd3b5dd3b96aad345